src/main/java/com/softserve/actent/model/entity/EventUser.java

```diff
@@ -0,0 +1,32 @@
+package com.softserve.actent.model.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "events_users")
+public class EventUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    private Event event;
```

Comment:
the default fetch type of @ManyToOne is EAGER. Do you need it here? If so, than explain why.

Reply:
It's temporary, after detail tests I can configure it!